distribution/docs/src/main/resources/content/_managing/_configuring/web-context-policy-manager.adoc

```diff
@@ -27,14 +27,33 @@ Note that the `SAML` and `OIDC` authentication types require session storage to
 
 === Authentication Types
 
-As you add REST endpoints, you may need to add different types of authentication through the Web Context Policy Manager.
+Through the Web Context Policy Manager, authentication types for ${branding} may be configured with the following format:
 
-Any web context that allows or requires specific authentication types should be added here with the following format:
+----
+<AUTH_TYPE>|<AUTH_TYPE|...
+----
+
+Authentication types can be configured separately for:
+* web pages (these are all context paths that do not start with `/services` such as `/admin` and `/search`)
+* endpoints (these are all context paths that start with `/services`)
```

Comment:
❓ What about endpoints that aren't under /services? I.e. Intrigue internal endpoints, the Jolokia endpoints, etc.

Reply:
Those are put under web pages
```
web pages (these are all context paths that do not start with `/services` such
as `/admin` and `/search`)
```

Reply:
If endpoints need to be handled differently, though, why only the endpoints under /services? I guess my confusion comes down to why we handle endpoints and web pages differently in the first place. https://github.com/codice/ddf/pull/6003#discussion_r411505169